Core/src/LongRunning/OperationResponseTrait.php

```diff
@@ -103,7 +107,42 @@ private function deserializeResult($operation, $type, $serializer, array $mapper
 
         $response = new $message();
         $anyResponse = $operation->getLastProtoResponse()->getResponse();
+        if (is_null($anyResponse)) {
+            return null;
+        }
+
+        $response->mergeFromString($anyResponse->getValue());
+
+        return $serializer->encodeMessage($response);
+    }
 
+    /**
+     * Convert an operation metadata to an array
+     *
+     * @param OperationResponse|GaxOperationResponse $operation The operation to
+     *        serialize.
+     * @param string $type The Operation type. The type should correspond to a
+     *        member of $mappers.typeUrl.
+     * @param Serializer|GaxSerializer $serializer The gRPC serializer to use
+     *        for the deserialization.
+     * @param array $mappers A list of mappers.
+     * @return array|null
+     */
+
+    private function deserializeMetadata($operation, $type, $serializer, array $mappers)
+    {
+        $mappers = array_filter($mappers, function ($mapper) use ($type) {
+            return $mapper['typeUrl'] === $type;
+        });
+        if (count($mappers) === 0) {
+            throw new \RuntimeException(sprintf('No mapper exists for operation response type %s.', $type));
```

Comment:
Why throw here? Seems more resilient to API changes to continue without doing anything if a mapper isn't found.

Reply:
Similar as in `deserializeResult()` function on line 101 above.
Since this logic is internal, I think it will help catch and update the code in accordance with the API changes